Jenkinsfile

```diff
@@ -1 +1,3 @@
+// Builds a module using https://github.com/jenkins-infra/pipeline-library
 buildPlugin()
```

Comment:
Something along `buildPlugin.recommendedConfigurations().filter { it.jenkinsVersion >= "2.165" }` should work I think.

Goal being: we'd have CI also on Windows.

Reply:
The default config runs CI on Windows. Just only against the default `jenkins.version` and using only JDK 8.